src/components/Toolbar.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { connect } from 'react-redux'
 import { shortcutById } from '../shortcuts'
+import { overlayReveal, overlayHide } from '../action-creators/toolbar'
```

Comment:
Great! Thanks for creating an action-creator for this.

Reply:
I didn't, someone else did.